nessie/jobs/generate_canvas_caliper_analytics.py

```diff
@@ -56,18 +55,10 @@ def run(self):
         if redshift_response:
             if redshift_response[0].get('count'):
                 app.logger.info('Generating user request tables and caliper analytics.')
-                redshift_schema_canvas = app.config['REDSHIFT_SCHEMA_CANVAS']
-                redshift_schema_caliper_analytics = app.config['REDSHIFT_SCHEMA_CALIPER']
 
-                resolved_ddl_caliper_analytics = resolve_sql_template(
-                    'generate_caliper_analytics.template.sql',
-                    redshift_schema_lrs_external=redshift_schema_lrs_external,
-                    redshift_schema_canvas=redshift_schema_canvas,
-                    redshift_schema_caliper_analytics=redshift_schema_caliper_analytics,
-                )
```

Comment:
Note that these values don't need to be parameterized since they're provided as defaults in nessie/lib/util.py, below. Elsewhere in this file, `resolve_sql_template` takes parameters that really do vary and continue to be passed in from code.